server/async_tcp.go

```diff
@@ -1,15 +1,11 @@
 package server
 
 import (
```

Comment:
Also, update the README and add OSX (Darwin) into the supported platforms (especially the place we have been explicit about Linux-based Development Environment).

Reply:
Updated the README. Please have a look.